libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java

```diff
@@ -1548,6 +1559,30 @@ public static String resetUploadingMediaToFailed(Context context, @NonNull Strin
         return postContent;
     }
 
+    public static List<String> getMediaMarkedUploadingInPost(Context context, @NonNull String postContent) {
```

Comment:
I'm being nitpicky, but this method seems to be misnamed since other methods that have `Post` is their name operate on a `PostModel`. How about `getMediaMarkedUploadingInPostContent`?